components/aks/aks.tf

```diff
@@ -1,11 +1,11 @@
 resource "azurerm_resource_group" "kubernetes_resource_group" {
-  count    = var.cluster_count
+  for_each = toset([for k, v in var.clusters : k])
```

Comment:
Do we want to make this more user readable?

Reply:
Sure thing, just followed SDS set up for now , but will rewrite